src/main/drivers/pwm_output_dshot_hal.c

```diff
@@ -36,63 +38,86 @@
 
 #include "pwm_output.h"
 
-typedef DMA_Stream_TypeDef dmaStream_t;
+// TODO remove once debugging no longer needed
```

Comment:
Is it possible to remove this before this goes into a release candidate?

Reply:
I think it's useful to keep this troubleshooting information around. Maybe I should add a new cli command like dshot_bidir_stats or so? I can add a real api for this and allow people to troubleshoot bidir since I expect we will need that option.

Reply:
All good, we can keep it in then for 4.0, and reconsider removing it at a later stage when the protocol is more stable.